app/views/published_pages/show.html.erb

```diff
@@ -1,18 +1,24 @@
-<div class="published-page-wrapper">
-  <div class="published-page-header">
-    <h1 class="published-page-title"><%= @topic.title %></h1>
-
+<div class="pp-header">
+    <div class="pp-header-wrapper">
+      <a href="<%= @site_url %>">
+        <img class="pp-logo" src="<%=@logo.url%>"/>
+      </a>
+      <h1 class="pp-title"><%= @topic.title %></h1>
+    </div>
+  </div>
+<div class="published-page-content-wrapper">
+  <div class="published-page-content-header">
     <div class="published-page-author">
-      <img width="45" height="45" src="<%= @topic.user.avatar_template.gsub('{size}', '90') %>" class="avatar">
+      <img width="45" height="45" src="<%= @topic.user.avatar_template.gsub('{size}', '90') %>" style="border: 4px solid <%= @border_color %>90" class="avatar">
```

Comment:
as I hate inline style, something we could at least do here is only having border-color :p and move width/style in css